src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  max-width: 200px;
+  min-height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
```

Comment:
The color (`#060b35`) is used multiple times. Consider defining it as an SCSS variable for consistency and easier maintenance.